### DISCUSSION TITLE
RFC 003 does not list source tool

### DISCUSSION THREAD

**Discussion by Dan33l** [puppetlabs/pdk#1004] [Feature Request]
In the [RFC 0003](https://github.com/puppetlabs/pdk-planning/blob/master/RFCs/0003-add-pdk-release.md), no source tool is listed as accepted with `pdk release` command.

With [PDK PR PR 809](https://github.com/puppetlabs/pdk/pull/809) a first step is provided with Github as source. But the RFC was not Github specific. It looks useful to talk about sources planed.

**Comment by scotje** [comment]
I think that requirement is coming from Github Changelog Generator which will be made an optional part of `pdk release` process in a future revision.
